WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController.m

```diff
@@ -141,7 +136,7 @@ - (void)viewDidLoad
     UIBarButtonItem *backButton = [[UIBarButtonItem alloc] initWithTitle:[NSString string] style:UIBarButtonItemStylePlain target:nil action:nil];
     self.navigationItem.backBarButtonItem = backButton;
     
-    [self updateTabBarBadgeNumber];
+    [[WPTabBarController sharedInstance] updateNotificationBadgeVisibility];
```

Comment:
I should have gotten rid of that one completely, `WPTabBarController` already handles the initiation.